<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Credits</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../images/openshot.qrc">
    <normaloff>:/openshot.svg</normaloff>:/openshot.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabCredits">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabDevelopers">
      <attribute name="title">
       <string>Developers</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="vboxDevelopers">
         <item>
          <layout class="QHBoxLayout" name="hboxDevelopers">
           <item>
            <widget class="QLineEdit" name="txtDeveloperFilter">
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>Filter Developers</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTranslators">
      <attribute name="title">
       <string>Translators</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="vboxTranslators">
         <item>
          <layout class="QHBoxLayout" name="hboxTranslators">
           <item>
            <widget class="QLineEdit" name="txtTranslatorFilter">
             <property name="placeholderText">
              <string>Filter Translators</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSupporters">
      <attribute name="title">
       <string>Supporters</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="vboxSupporters">
         <item>
          <layout class="QHBoxLayout" name="hboxSupporters">
           <item>
            <widget class="QLineEdit" name="txtSupporterFilter">
             <property name="placeholderText">
              <string>Filter Supporters</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblBecomeSupporter">
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>258</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnclose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnclose</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
